variants/Generic_F401Rx/variant.h

```diff
@@ -0,0 +1,171 @@
+#define PB7  23 // |        | USART1_RX | TWI1_SDA |                        |           |
+#define PB8  24 // |        |           | TWI1_SCL |                        |           |
+#define PB9  25 // |        |           | TWI1_SDA | SPI2_SS                |           |
+#define PB10 26 // |        |           | TWI2_SCL | SPI2_SCK               |           |
+#define PB12 27 // |        |           |          | SPI2_SS                |           |
+#define PB13 28 // |        |           |          | SPI2_SCK               |           |
+#define PB14 29 // |        |           |          | SPI2_MISO              |           |
+#define PB15 30 // |        |           |          | SPI2_MOSI              |           |
+//                 |--------|-----------|----------|------------------------|-----------|
+#define PC0  31 // | A10    |           |          |                        |           |
+#define PC1  32 // | A11    |           |          |                        |           |
+#define PC2  33 // | A12    |           |          | SPI2_MISO              |           |
+#define PC3  34 // | A13    |           |          | SPI2_MOSI              |           |
+#define PC4  35 // | A14    |           |          |                        |           |
+#define PC5  36 // | A15    |           |          |                        |           |
+#define PC6  37 // |        | USART6_TX |          |                        |           |
+#define PC7  38 // |        | USART6_RX |          |                        |           |
+#define PC8  39 // |        |           |          |                        |           |
+#define PC9  40 // |        |           | TWI3_SDA |                        |           |
+#define PC10 41 // |        |           |          | SPI3_SCK               |           |
+#define PC11 42 // |        |           |          | SPI3_MISO              |           |
+#define PC12 43 // |        |           |          | SPI3_MOSI              |           |
+#define PC13 44 // |        |           |          |                        |           |
+#define PC14 45 // |        |           |          |                        | OSC32_IN  |
+#define PC15 46 // |        |           |          |                        | OSC32_OUT |
+//                 |--------|-----------|----------|------------------------|-----------|
+#define PD2  47 // |        |           |          |                        |           |
+//                 |--------|-----------|----------|------------------------|-----------|
+#define PH0  48 // |        |           |          |                        | OSC_IN    |
+#define PH1  49 // |        |           |          |                        | OSC_OUT   |
+//                 |--------|-----------|----------|------------------------|-----------|
+
+// This must be a literal
+#define NUM_DIGITAL_PINS        50
+#define NUM_ANALOG_INPUTS       16
+#define NUM_ANALOG_FIRST        0
+#define AEND                    36
```

Comment:
I guess this could be removed as AEND is no more used in Firmata.

Reply:
OK. I'll remove AEND. But how should all these small changes be committed? One commit for each file? One commit for everything?

Reply:
Well for example in this case:
`AEND` has been added to this new file, so simply remove it from the commit which add it.
For the `AEND `in the `pin_arduino.h` then a single commit is a good idea and set a proper commit message to explain that it is removed as Firmata does not use it anymore.

Reply:
So one separate commit that removes AEND from pins_arduino.h and variant.h with a proper description that describes why the are being removed?

Reply:
> So one separate commit that removes AEND from pins_arduino.h ~and variant.h~ with a proper description that describes why they are being removed?

For variant.h this have to be in the same commit than 93a67409eed6c9e620e93bfd58b604cabc6e7c90

Reply:
Small comments all around. I'll sum it up in a separate comment when we have figured this out before I start fixing these small issues. At this point, I'm too afraid of pushing something that might pollute the history...